wgpu-hal/src/gles/adapter.rs

```diff
@@ -186,10 +186,20 @@ impl super::Adapter {
         let extensions = gl.supported_extensions();
 
         let (vendor_const, renderer_const) = if extensions.contains("WEBGL_debug_renderer_info") {
+            // context must be current
```

Comment:
Comment in the wrong place

Reply:
where should i place it? i thought being closest the unsafe usage would be good.